core/services/chainlink/config.go

```diff
@@ -70,6 +71,16 @@ func (c *Config) valueWarnings() (err error) {
 			}
 		}
 	}
+	// ChainType xdai is deprecated and has been renamed to gnosis
+	for _, evm := range c.EVM {
+		if evm.ChainType != nil && *evm.ChainType == string(commoncfg.ChainXDai) {
+			err = multierr.Append(err, config.ErrInvalid{
+				Name:  "EVM.ChainType",
+				Value: *evm.ChainType,
+				Msg:   "deprecated and will be removed in v2.12.0, use 'gnosis' instead",
+			})
+		}
+	}
```

Comment:
Let's move this to `deprecatedWarnings`

Reply:
My proposal would be to skip condition `*evm.ChainType == string(commoncfg.ChainXDai)` and display the warning/s about future deprecations in any case when the node starts, no matter whether a user's specs are associated with the `xdai` chain.

Reply:
@jmank88 I moved the check to `deprecationsWarnings()` but kept it as a `ErrInvalid` instead of `ErrDeprecated` as I thought the latter is more suited to entire fields being deprecated. Does that work for you?

Reply:
I think that is probably OK